broker/src/asterisk/ami_client.erl

```diff
@@ -60,7 +60,9 @@ handle_call(_Request, _From, State) ->
 
 %% @private
 handle_cast(connect, State = #state{connected = false}) ->
-  case gen_tcp:connect("localhost", 5038, [binary, {packet, line}], 1000) of
+  AmiHost = application:get_env(verboice, ami_host, "localhost"),
+  AmiPort = application:get_env(verboice, ami_port, 5038),
```

Comment:
I would move this configuration to the `gen_server` initialization so it can be provided from outside the process itself.